api/notd/model.py

```diff
@@ -131,6 +131,22 @@ class Block:
     blockHash: str
     blockDate: datetime.datetime
 
+@dataclasses.dataclass
+class RetrievedTokenOwnership:
+    ownerAddress: str
+    registryAddress: str
+    tokenId: str
+    purchasedDate: datetime.datetime
+    purchasedValue: int
+    transferId: int
```

Comment:
ah sorry i would remove this - we delete transfers quite often now because of the reprocessing